packages/admin/src/middleware/Auth.middleware.ts

```diff
@@ -17,6 +17,7 @@ export function getAuthMiddleware(grpcSdk: ConduitGrpcSdk, conduit: ConduitCommo
       req.originalUrl.indexOf('/graphql') === 0 && req.method === 'GET';
     if (
```

Comment:
can we move the exclusions to an array and add a cleaner check? It's getting too big at this point.